pages/index.js

```diff
@@ -15,7 +15,7 @@ export default function Home() {
 
       <ul>
         {recipes?.map((recipe) => (
-          <FoodCard recipe={recipe} />
+          <FoodCard recipe={recipe} key={recipes.id} />
```

Comment:
`recepies.id` could be a problem (the plural form) try the passed `recipe` value ;-)